packages/mold.rb

```diff
@@ -44,9 +44,14 @@ def self.build
         -Wno-dev \
         -G Ninja"
     system "#{CREW_NINJA} -C builddir"
+    File.write 'moldenv', <<~MOLD_ENV_EOF
+      # See https://github.com/rui314/mold/commit/36fc0655489eb96e1be15b03b3f5e227cd97a22e
+      MOLD_JOBS=1
+    MOLD_ENV_EOF
   end
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir install"
+    FileUtils.install 'moldenv', "#{CREW_DEST_PREFIX}/etc/env.d/mold", mode: 0o644
   end
```

Comment:
Add?
```
def self.postinstall
  puts "\nTo finish the installation, execute the following:".lightblue
  puts "source ~/.bashrc\n".lightblue
end
```
Otherwise, the environment variable is not sourced?

Reply:
It is sourced on next crosh invocation, like every single other env.d file...

Reply:
postinstall message added.